kernel.go

```diff
@@ -95,6 +95,12 @@ func runKernel(connectionFile string) {
 	ir.Comp.Stdout = ioutil.Discard
 	ir.Comp.Stderr = ioutil.Discard
 
+	// Inject the "display" package to render HTML, JSON, PNG, JPEG, SVG... from interpreted code
+	_, err := ir.Comp.ImportPackageOrError("display", "display")
+	if err != nil {
+		log.Print(err)
+	}
+
```

Comment:
Something like in f696d7632076934f53d928ec5fe4cee5c222fb64 that replaces the display function with a closure that knows about the eval context removes the global. Not sure of any repercussions gomacro will have as a result but taking advantage of the more dynamic interpreter seemed like a great use for gomacro.